lib/modelMaterialsCommon.js

```diff
@@ -0,0 +1,754 @@
+"use strict";
+var Cesium = require('cesium');
+var WebGLConstants = Cesium.WebGLConstants;
+var defined = Cesium.defined;
+var defaultValue = Cesium.defaultValue;
+
+module.exports = modelMaterialsCommon;
+
+function webGLConstantToGlslType(webGLValue) {
```

Comment:
Cesium has its own `modelMaterialsCommon` with some implementation differences, so I'm guessing the change would be better happening over there.